docs/contributors/code/getting-started-with-code-contribution.md

```diff
@@ -2,47 +2,65 @@
 
 We recommend using the [Node Version Manager](https://github.com/nvm-sh/nvm) (nvm) since it is the easiest way to install and manage node for macOS, Linux, and Windows 10 using WSL2. See [our Development Tools guide](/docs/getting-started/tutorials/devenv/README.md#development-tools) or the Nodejs site for additional installation instructions.
 
-After installing Node, you can build Gutenberg by running the following from within the cloned repository:
+- Git
+Gutenberg is using git for source control. Make sure you have an updated version of git installed on your computer, as well as a GitHub account. You can read the [Git Workflow](/docs/contributors/code/git-workflow.md) to learn more about using git and GitHub with Gutenberg
 
+- [Optional] Docker Desktop
+We recommend using the [wp-env package](/packages/env/README.md) for setting WordPress environment locally. You'll need to install Docker to use `wp-env`. See the [Development Environment tutorial for additional details](/docs/getting-started/tutorials/devenv/README.md).
+> Note: To install Docker on Windows 10 Home Edition, follow the [install instructions from Docker for Windows with WSL2](https://docs.docker.com/docker-for-windows/wsl/).
+
+## Getting the Gutenberg code
+Fork the Gutenberg repository, clone it to your computer and add the WordPress repository as upstream.
 
 ```bash
-npm ci
-npm run build
+$ git clone https://github.com/YOUR_GITHUB_USERNAME/gutenberg.git
+$ cd gutenberg
+$ git remote add upstream https://github.com/WordPress/gutenberg.git
 ```
 
+## Building Gutenberg as a plugin
+Install the Gutenberg dependencies and build your code in development mode:
+
+```bash
+npm ci
+npm run dev
+```
 > Note: The install scripts require [Python](https://www.python.org/) to be installed and in the path of the local system. This might be installed by default for your operating system, or require downloading and installing.
 
-Once built, Gutenberg is ready to be used as a WordPress plugin!
+There are two ways to build your code. While developing, you probably will want to use `npm run dev` to run continuous builds automatically as source files change. The dev build also includes additional warnings and errors to help troubleshoot while developing. Once you are happy with your changes, you can run `npm build` to create optimized production build.
 
-`npm run build` creates a single build of the project once. While developing, you probably will want to use `npm run dev` to run continuous builds automatically as source files change. The dev build also includes additional warnings and errors to help troubleshoot while developing.
+Once built, Gutenberg is ready to be used as a WordPress plugin!
 
-## Local Environment
+## Local WordPress Environment
 
 If you are familiar with WordPress and already have an environment setup, use the above build as a standard WordPress plugin by putting the gutenberg directory in your wp-content/plugins/ directory.
```

Comment:
To me it is not very clear what 'Local WordPress Environment setup'. I have installed WordPress downloaded from wordpress.org before. But 'Local WordPress Environment' sounds like it's something quite different, and I'm not sure _what_ the differences are. So, maybe adding the sentence from below in this section : 'To test (or try out) a WordPress plugin (or Gutenberg WordPress plugin), you need to have WordPress installed' would make it clearer?